### ISSUE TITLE
Is partial update available for Azure Cosmos DB v4 sdk?

### ISSUE THREAD

**Issue by chialiyun** [AzureCosmosDB/PatchPrivatePreviewFeedbackGroup#28]
https://docs.microsoft.com/en-us/azure/cosmos-db/sql/create-sql-api-dotnet-v4

The .NET cosmosdb v4 is using Azure.Cosmos --version 4.0.0-preview3 package.

**Comment by sajeetharan** [comment]
No, there are no plans to work on V4 , please use V3 for any purposes
